README.md

```diff
@@ -40,20 +44,28 @@ The baseline for this dataset is Molecule Attention Transformer, fine-tuned with
 The baseline for this dataset is Audio Spectrogram Transformer, more details can be found in [paper](https://arxiv.org/abs/2104.01778).
 
 ## Question 1
-Can pretrained language models transfer to different modalities?
+* Can pretrained language models transfer to different modalities?
+* Only input embedding layer, output readout layer and layer norm parameters are fine-tuned.
+* FPT archives comparable performance to the fully trained transformer.
+* Models are learning representations and performing computation that is agnostic to the modality.
```

Comment:
I'm not really sure. Adjusting learned representations to a specific task is the point of fine-tuning AFAIK.

Reply:
I guess is duplicate comment.

Reply:
True